package.json

```diff
@@ -145,10 +146,11 @@
     "eslint-plugin-import": "^2.29.1",
     "eslint-plugin-jsx-a11y": "^6.9.0",
     "eslint-plugin-prettier": "^5.0.0",
-    "eslint-plugin-react": "7.33.2",
+    "eslint-plugin-react": "^7.37.1",
     "eslint-plugin-react-hooks": "4.6.0",
     "file-loader": "^6.2.0",
     "fs-extra": "11.2.0",
+    "globals": "^15.11.0",
```

Comment:
what does this package do?

Reply:
probably was added when installing and trying to fix issue with eslint. Will try to clean it up